src/DataProtection/DataProtection/src/Managed/ManagedAuthenticatedEncryptor.cs

```diff
@@ -187,8 +193,6 @@ public byte[] Decrypt(ArraySegment<byte> protectedPayload, ArraySegment<byte> ad
             }
 
             ArraySegment<byte> keyModifier = new ArraySegment<byte>(protectedPayload.Array!, keyModifierOffset, ivOffset - keyModifierOffset);
-            var iv = new byte[_symmetricAlgorithmBlockSizeInBytes];
```

Comment:
just want to check: this is the main reduction (on suitable TFMs), yes?

Reply:
there are a couple of `byte[]` allocations I am removing. I am going to explain it in details in the PR description. I will resolve comment once I have written the PR description